frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -70,30 +68,30 @@ export default function WorkflowListView() {
   useEffect(() => {
     const { searchQuery } = persistence.load();
     setSearchQuery(searchQuery || "");
-    loadData();
   }, []);
 
   useEffect(() => {
     persistence.persist({ searchQuery });
   }, [searchQuery]);
 
-  async function loadData() {
-    setIsLoading(true);
-    try {
-      const _workflows = (await getVoxelyticsWorkflows()).map(parseWorkflowInfo);
-      setWorkflows(_workflows);
-    } catch (err) {
-      Toast.error("Could not load workflow list.");
-      console.error(err);
-    } finally {
-      setIsLoading(false);
-    }
-  }
-
-  usePolling(async () => {
-    // initial data fetch is done above, thus only load data here if it is polled repeatedly
-    if (VX_POLLING_INTERVAL != null) loadData();
-  }, VX_POLLING_INTERVAL);
+  const {
+    data: workflows = [],
+    isLoading,
+    refetch,
+    isFetching,
+  } = useQuery({
+    queryKey: ["voxelyticsWorkflows"],
+    queryFn: async () => {
+      try {
+        return (await getVoxelyticsWorkflows()).map(parseWorkflowInfo);
+      } catch (err) {
+        Toast.error("Could not load workflow list.");
+        console.error(err);
+        throw err;
+      }
+    },
+    refetchInterval: VX_POLLING_INTERVAL ?? 0,
```

Comment:
according to the docs, this should be: `refetchInterval: number | false | ((query: Query) => number | false | undefined)`. so, passing 0 as a fallback sounds a bit dangerous. can you use `?? false`?